WebRoot/css/zm.css

```diff
@@ -1975,14 +1975,48 @@ div.CompDragArea * {
 	display:inline-block;
 }
 
-.newTaskBanner {
-	@InactiveCursor@
-	color:rgb(120, 111, 91);
-	padding:5px 0px;
+div.newTaskBanner div.ImgAdd-svg {
+	@TaskListViewAddNewTaskIcon@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-ColHeader {
+	border-bottom:0px;
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble {
+	@TaskListViewItem@
+}
+
+.ZmTaskMultiView .DwtListView .RowDouble td {
+	@TaskListViewItemCheckbox@
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble .TaskSubject {
+	@TaskListViewItemSubject@
 }
 
-.newTaskBannerSep {
-	border-bottom:2px solid #AAA;
+.ZmTaskMultiView .DwtListView  .RowDouble .TaskDate {
+	@TaskListViewItemDate@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow {
+	@TaskListViewItemBottomContainer@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow .IconOuterWrapper {
+	@TaskListViewItemBottomIconOuterWrapper@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow .IconInnerWrapper {
+	@TaskListViewItemBottomIconInnerWrapper@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-Rows  .ImgCheckboxUnchecked-svg {
+	padding-top:32px;
+}
+
+.ZmTaskMultiView .DwtListView  .Row-selected .ImgCheckboxChecked-svg {
+	padding-top:32px;
```

Comment:
Had I been to implement circular progress bar component, I would have put  that in another TD of the row, which is independent from the td which is containing checkbox. And, this will not break things.